src/System.Text.Json/tests/JsonTestHelper.cs

```diff
@@ -17,6 +17,14 @@ namespace System.Text.Json
 {
     internal static class JsonTestHelper
     {
+#if BUILDING_INBOX_LIBRARY
+        public const string DoubleFormatString = "G";
```

Comment:
These make it easier to normalize the format difference between full framework and core elsewhere.

Reply:
Should the StandardFormat values be here as static readonly, just so all of the format variance is in one contained place?

Reply:
No longer relevant.